src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -97,52 +97,78 @@ internal class BindingProcessImpl : IBindingProcess
 
         public async Task<bool> DownloadQualityProfileAsync(IProgress<FixedStepsProgress> progress, CancellationToken cancellationToken)
         {
-            var languageList = this.GetBindingLanguages();
```

Comment:
Miscellaneous: removed `this.` from the changed method to get rid of some code smells. I didn't do it for the whole file in this PR as that would have been too noisy.